Public/New-DirectoryBrowser.ps1

```diff
@@ -0,0 +1,66 @@
+function New-DirectoryBrowser {
+
+    <#
+    .SYNOPSIS
+        Renders a directory browser in HTML
+    .DESCRIPTION
+        Creates HTML that can be used as a directory browser
+    .PARAMETER RequestedItem
+        The directory you would like to generate HTML for
+    .PARAMETER HeaderName
+        The name you would like displayed at the top of the directory browser
+    .PARAMETER DirectoryBrowserPath
+        The current path in the directory browser relative to the root of the directory
+        browser (not the root of the site).
+#>
+
+    param (
+
+        [Parameter(
+            Mandatory = $true,
+            HelpMessage = 'Directory Path')]
+        [System.IO.DirectoryInfo]$RequestedItem,
+
+        [Parameter(
+            Mandatory = $false,
+            HelpMessage = 'Header Name')]
+        [string]$HeaderName = "Polaris Directory Browser",
+
+        [Parameter(
+            Mandatory = $false,
+            HelpMessage = 'Directory Browser Path')]
+        [string]$DirectoryBrowserPath
+    )
+
+    Write-Debug "DirectoryBrowserPath: $DirectoryBrowserPath"
+
+    @"
+<html>
+<head>
+<title>$HeaderName</title>
+</head>
+<body>
+<h1>$HeaderName - $DirectoryBrowserPath</h1>
+<hr>
+$(if ($RequestedItem.FullName.TrimEnd("\") -ne $RequestedItem.PSDrive.Root) { '<a href="./../">[To Parent Directory]</a><br><br>'})
```

Comment:
On my mac, paths use `/` instead of `\`. Might want to add a `.TrimEnd("/")` as well or use:
`[System.IO.Path]::DirectorySeparatorChar`

(not sure if that's supported in Windows PowerShell though.

Reply:
It is, good call.